.github/workflows/mediator.pr.yml

```diff
@@ -0,0 +1,56 @@
+name: CI mediator
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "**"
+    paths:
```

Comment:
didn't know about this, pretty cool 👍 answers my just-about-to-ask question why is this a separate file